webextensions/api/browserSettings.json

```
@@ -113,7 +113,8 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": "72"
+                "version_added": "72",
+                "notes": "From Firefox version 88 this setting is read-only (<a href='https://bugzil.la/1363010'>bug 1626365</a>)."
```

A few things here:

- There's a mismatch here between the bug text number and URL. This is the cause of the CI failure. I'm assuming you meant the text, not the URL, but please double check this.
- Generally, prefer "version XY" or "Browser XY" over "Browser version XY".
- Other nitpicky things about commas and referring to bug links I would probably wouldn't have otherwise bothered with, but since I'm already here…

```suggestion
                "notes": "From version 88, this setting is read-only (see <a href='https://bugzil.la/1626365'>bug 1626365</a>)."
```